### ISSUE TITLE
Registration page should display error message if user tries to register a duplicate account

### ISSUE THREAD

**Issue by rstorey** [LibraryOfCongress/concordia#1127]
**What behavior did you observe? Please describe the bug**
The register page refreshed. The password and confirm password fields emptied.

**How can we reproduce the bug?**

1. Go to the Register page
2. Fill out the form.
3. Go back to a blank Register page.
4. Fill out the form using the same username or email address as in the first try

**What is the expected behavior?**
If a user attempts to register either a username or email address that is already in use by another account, an error message should be displayed and the user should be prompted to use a different username / email address. Perhaps there should also be a link to the reset password page.

**Desktop (please complete the following information):**

-   OS: MacOS
-   Browser Chrome

**Comment by acdha** [comment]
The password fields employing is a security measure. This used to work and the form sets the invalid state on the email field so this was probably a template change suppressing display of the error message for that field since we chose not to use the standard Django forms.

**Comment by rstorey** [comment]
Yep. I only mentioned the password thing because that was the only way I could tell the page actually refreshed.

**Comment by rstorey** [comment]
https://github.com/twbs/bootstrap/issues/23454

**Comment by rstorey** [comment]
https://github.com/twbs/bootstrap/issues/29439
